Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/Util.cs

```diff
@@ -1,51 +1,51 @@
 ﻿using System.Text.RegularExpressions;
 
-namespace Frends.SFTP.UploadFiles.Definitions
+namespace Frends.SFTP.UploadFiles.Definitions;
+
+/// <summary>
+/// Helper methods for file modifications
+/// </summary>
+internal static class Util
 {
-    /// <summary>
-    /// Helper methods for file modifications
-    /// </summary>
-    internal static class Util
+    internal static string CreateUniqueFile(string toDir)
     {
-        internal static string CreateUniqueFile(string toDir)
+
+        if (!toDir.EndsWith("/") && !toDir.EndsWith("\\"))
         {
+            toDir = toDir + "\\";
```

Comment:
You can use `toDir += "\\"` here. Also, one-line if.

Reply:
Fixed